articles/reporting/metrics-definitions.md

```diff
@@ -159,3 +159,11 @@ Number of messages opened or have at least one link clicked.
 **Unique Rendered (approximated)**
 
 Number of unique recipients who open a message.
+
+**Unsubscribes**
+
+Total number of times a recipient clicked a specifically tagged unsubscribe link or unsubscribed by using the ISP list unsubscribe feature.
+
+**Unsubscribe Rate**
+
+The percentage messages that resulted in unsubscribes. Calculated by _number of all unsubscribes / number of messages sent._
```

Comment:
number of messages sent or accepted?

Reply:
Accepted, but I didn't see that in any of the existing `Calculated by` notes. Used sent for consistency. However, some rates are missing the calculation alltogether...